src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1337,4 +1367,71 @@ private class ACCT_IndividualAccounts_TEST {
         
         return newRecTypeId;
     }
+
+    /*********************************************************************************************************
```

Comment:
since you repeat all of these test helper methods in multiple test classes, i'd prefer seeing them in one of the HH test classes, and just made public so the other test classes can use them.  that's a paradigm we've used in other components.